djangoUImanager/Web/migrations/0001_initial.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.18 on 2024-04-25 13:12
```

Comment:
You should be able to flatten your migrations here into one (seems you have 13 here)